packages/framework/src/Migrations/DataModifiers/CountryDataModifierVersion20190121094400.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Migrations\DataModifiers;
+
+use function array_diff;
+use function array_key_exists;
+use function array_keys;
+use function array_values;
+
+class CountryDataModifierVersion20190121094400
+{
+    /**
+     * @var array
+     */
+    private $data;
+
+    /**
+     * @var array
+     */
+    private $tmpIds;
+
+    /**
+     * @param array $data
+     */
+    public function __construct(array $data)
+    {
+        $this->data = $data;
+        $this->tmpIds = [];
+    }
+
+    /**
+     * @return array
+     */
+    public function getGroupedByCode(): array
+    {
+        $tmp = [];
+        foreach ($this->data as $row) {
+            $tmp[$row['code']][] = $row;
+        }
+
+        return $tmp;
+    }
+
+    /**
+     * @return array
+     */
+    public function getNewIdCodePair(): array
+    {
+        $tmp = [];
+        foreach ($this->data as $row) {
```

Comment:
This function is too complex, data here should be ordered by domain_id so you could remove first foreach and remover $row['domain_id] === 1 from condition

Reply:
I simplified the function. Please check if it's ok.